code/nathan/python/contact_list.csv

```diff
@@ -0,0 +1,4 @@
+Name,Phone Number,Email
+Nathan,555-555-5555,nathan@fake.com
+Jimmy,777-777-7777,jimmy@fake.com
+John Dillon,123-456-7891,jdillon@gmail.com
```

Comment:
Apparently an empty line at the end of your csv file can create errors in your program because its looking for a key that does not exist. Who knew? (Matt knew) The more you know.